src/api/app/models/bs_request/find_for/group.rb

```diff
@@ -4,24 +4,39 @@ class Group < Base
       include UserGroupMixin
 
       def all
-        inner_or = []
-        group = ::Group.find_by_title!(group_title)
+        query = union_query
+        @relation = if query.present?
+                      @relation.where("bs_requests.id IN (#{query})")
+                    else
+                      @relation.none
+                    end
+      end
+
+      def all_count
+        ActiveRecord::Base.connection.execute("SELECT COUNT(bs_request_id) FROM (#{union_query}) x").first.first
```

Comment:
What is the double `.first` for?

Reply:
It returns a list of rows. We want the first element of the first row.